### ISSUE TITLE
AMO

### ISSUE THREAD

**Issue by wvxwxvw** [airtower-luna/referer-mod#52]
On AMO is no Russian localizatio

**Comment by airtower-luna** [comment]
That's true, I don't have a Russian version of the listing. If you're willing to create and maintain one I'm happy to upload it. By "maintain" I mean update if there are substantial changes to the English version.

**Comment by airtower-luna** [comment]
I just noticed the Russian local file is missing in the uploaded version. Did you mean that?

**Comment by airtower-luna** [comment]
Sorry I missed that! I've just pushed an update (version 0.12.1, commit 9aae53f278b569281f81eefd7f4496dde0196941), if you update to that the Russian localization should be there.

**Comment by wvxwxvw** [comment]
Thanks. I thought AMO was a common abbreviation addons.mozilla.org

**Comment by airtower-luna** [comment]
It is, but I didn't understand you meant the released version of the add-on, I thought you meant the description on AMO.
